pilot/pkg/config/kube/crd/codegen/types.go

```diff
@@ -60,7 +60,7 @@ func main() {
 	typeList := []ConfigData{
 		MakeConfigData(collections.Mock),
 	}
-	for _, s := range collections.Pilot.All() {
+	for _, s := range collections.PilotServiceApi.All() {
```

Comment:
do we need another collection for these? Can we not just include them in the Pilot one?

Reply:
The motivation was to allow us to easily ignore the service APIs by default, so that its truly "off by default" since there was some push back on the initial PR